README.md

```diff
@@ -11,14 +11,20 @@
   <br />
   <br />
   <a href="https://circleci.com/gh/FormidableLabs/urql">
-    <img alt="Test Status" src="https://circleci.com/gh/FormidableLabs/urql.svg?style=shield" />
-  </a> <a href="https://github.com/FormidableLabs/urql#maintenance-status">
-    <img alt="Maintenance Status" src="https://img.shields.io/badge/maintenance-active-green.svg" />
+    <img alt="Test Status" src="https://badgen.net/github/status/FormidableLabs/urql/master/ci/circleci?label=circleci" />
+  </a>
+  <a href="https://www.npmjs.com/package/urql">
+    <img alt="Total downloads" src="https://badgen.net/npm/dt/urql?color=blue" />
```

Comment:
I'm not entirely sure how valuable total downloads is as compared to monthly. Personally not a huge fan of adding the downloads in there

Reply:
Yea, it comes across as gloating at times. Which is fine at times I believe, we can celebrate our successes 😅 but I agree that monthly/weekly downloads are better

Reply:
I think total downloads is a nice metric for tracking milestones

Reply:
```suggestion
    <img alt="Total downloads" src="https://badgen.net/npm/dm/urql?color=blue" />
```